extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -511,11 +511,11 @@
 # Or:
 #     array_merge( $smwgPageSpecialProperties, array( '_CDAT' ) );
 # Or rewrite entire array:
-#     $GLOBALS['smwgPageSpecialProperties'] = array( '_MDAT', '_CDAT' );
+#     $smwDefaults['smwgPageSpecialProperties'] = array( '_MDAT', '_CDAT' );
 # However, DO NOT use `+=' operator! This DOES NOT work:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)